packages/reports/app/styles/navi-reports/common/report.scss

```diff
@@ -9,33 +9,29 @@
   flex-flow: column;
   max-width: 100%;
   min-height: 0;
-  margin-top: 10px;
 
-  &__breadcrumb {
-    margin: 5px 0 0 0;
-
-    &-link {
-      color: $color-brand-300;
+  .report-header {
+    &__breadcrumb {
+      &-link {
+        color: $color-brand-300;
+      }
     }
-  }
-
-  &__title {
-    display: flex;
-    font-size: $h1-font-size;
-    text-transform: capitalize;
-  }
 
-  &__breadcrumb,
-  &__header {
-    padding: 0 20px 0 25px;
-  }
+    &__title {
+      display: flex;
+      font-size: $h1-font-size;
+      text-transform: capitalize;
+    }
 
-  &__header {
-    align-items: center;
-    display: flex;
-    justify-content: space-between;
-    margin-bottom: 10px;
-    min-height: min-content;
+    &__header-action {
+      display: flex;
```

Comment:
```suggestion
      display: flex;
      button {
        white-space: nowrap;
      } 
```
This ^^ should fix this: 

![Screen Shot 2021-03-16 at 7 11 39 PM](https://user-images.githubusercontent.com/2983409/111395731-867f5500-868b-11eb-92c7-f592cb3d3905.png)